gdbstub_arch/src/riscv/mod.rs

```diff
@@ -15,21 +15,108 @@ pub enum Riscv64 {}
 impl Arch for Riscv32 {
     type Usize = u32;
     type Registers = reg::RiscvCoreRegs<u32>;
-    type RegId = reg::id::RiscvRegId<u32>;
     type BreakpointKind = usize;
+    type RegId = reg::id::RiscvRegId<u32>;
 
     fn target_description_xml() -> Option<&'static str> {
-        Some(r#"<target version="1.0"><architecture>riscv:rv32</architecture></target>"#)
+        // Source: https://github.com/bminor/binutils-gdb/blob/master/gdb/features/riscv/32bit-cpu.xml
+        // <!-- Copyright (C) 2018-2024 Free Software Foundation, Inc.
+        //
+        // Copying and distribution of this file, with or without modification,
+        // are permitted in any medium without royalty provided the copyright
+        // notice and this notice are preserved.  -->
+        Some(r#"<target version="1.0">
+<architecture>riscv:rv32</architecture>
```

Comment:
There's quite some duplication, but we can't use replace() of format! here due to require 'static string.

If preferred, I can attempt a macro solution

Reply:
Instead of hard-coding the string inline with the Rust code, can we just yoink the relevant XML files from GDB, put them alongside this `mod.rs` file, and use `include_str!` to reference them from the Rust code?

Reply:
Sounds good, will do!